git-flow/README.md

```diff
@@ -6,12 +6,12 @@
 We use a [**GitFlow**](https://nvie.com/posts/a-successful-git-branching-model/) branching model, where the repository holds two main **branches** with an infinite lifetime:
 
 - [**`master`**](https://github.com/miraisolutions/SmaRP/tree/master) reflects the [**latest release**](https://github.com/miraisolutions/SmaRP/releases/latest) to production, i.e. the current version of the [live app](https://mirai-solutions.ch/gallery/smarp).
-- [**`develop`**](https://github.com/miraisolutions/SmaRP/tree/develop) collects all completed developments for the [**next release**](#versioning-and-releases).
+- [**`develop`**](https://github.com/miraisolutions/SmaRP/tree/develop) is set as the default branch and collects all completed developments for the [**next release**](#versioning-and-releases).
 
-The overall GitFlow branching system is described as follows
+The overall GitFlow branching system is described as follows:
 
 - No work is committed and pushed directly to `master`, which is updated only as part of a [**release**](#versioning-and-releases).
-- Small (maintenance) work can be done directly in `develop`, however meaningful pieces should be developed in a dedicated **_feature_ branch** created from `develop` and associated to a GitHub issue (`<ID>`).
+- Only small maintenance work in context of releases can be done directly in `develop` (i.e. only version and `NEWS.md` changes, see also below). Meaningful pieces should be developed in a dedicated **_feature_ branch** created from `develop` and associated to a GitHub issue (`<ID>`).
```

Comment:
This was never meant to refer to the release but for small things. Still, it is worth mentioning that release preps can happen in develop, deferring to the dedicated session for details.

```suggestion
- Only small maintenance work, including [release preparation](#versioning-and-releases), can be done directly in `develop`. Meaningful pieces should be developed in a dedicated **_feature_ branch** created from `develop` and associated to a GitHub issue (`<ID>`).
```